webapp-django/crashstats/crashstats/models.py

```diff
@@ -987,11 +987,19 @@ def get(self, bugs):
             url = settings.BZAPI_BASE_URL + (
                 '/bug?id=%(bugs)s&include_fields=%(fields)s' % params
             )
-            response = requests_retry_session().get(
+            response = requests_retry_session(
+                # BZAPI isn't super reliable, so be extra patient
+                retries=5,
+                # 502 = Bad Gateway
+                # 504 = Gateway Time-out
+                status_forcelist=(500, 502, 504),
+            ).get(
                 url,
                 headers=headers,
                 timeout=self.BUGZILLA_REST_TIMEOUT,
             )
+            # Better fail here than getting a JsonDecodeError on the next line.
+            assert response.status_code == 200, response.status_code
```

Comment:
I'd make this assert message more informative. Maybe something like:

```
assert response.status_code == 200, (
    'Got non-200 status code: %s' % response.status_code
)
```

Otherwise it just has a number and it's much harder to figure out what's going on without tracking down the code that kicked it up.